pallets/ajuna-awesome-avatars/src/tests.rs

```diff
@@ -626,12 +638,17 @@ mod minting {
 
 	#[test]
 	fn mint_should_return_error_when_minting_is_unavailable() {
-		ExtBuilder::default().mint_availability(false).build().execute_with(|| {
-			assert_noop!(
-				AwesomeAvatars::mint(Origin::signed(ALICE), MintCount::One),
-				Error::<Test>::MintUnavailable
-			);
-		});
+		ExtBuilder::default()
+			.mint_availability(false)
+			.balances(vec![(ALICE, 1_000_000_000_000_000u64)])
+			.build()
+			.execute_with(|| {
+				assert_noop!(
+					AwesomeAvatars::mint(Origin::signed(ALICE), MintCount::One),
+					Error::<Test>::MintUnavailable
+				);
+				assert_eq!(Balances::total_balance(&ALICE), 1_000_000_000_000_000u64);
+			});
```

Comment:
Let's revert this back to the original test:
```suggestion
		ExtBuilder::default().mint_availability(false).build().execute_with(|| {
			assert_noop!(
				AwesomeAvatars::mint(Origin::signed(ALICE), MintCount::One),
				Error::<Test>::MintUnavailable
			);
		});
```

Reply:
Done